src/main/java/core/basesyntax/util/filewriter/FileWriterImpl.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.util.filewriter;
+
+import java.io.IOException;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String resultingReport, String toFileName) {
+        try (java.io.FileWriter fileWriter = new java.io.FileWriter(toFileName)) {
+            fileWriter.write(resultingReport);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
```

Comment:
The catch block should include an informative message along with the exception object. For example: throw new RuntimeException("Cannot write to file: " + toFileName, e);